Main.java

```diff
@@ -9,7 +10,30 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    // create a Scanner for user input
+    Scanner input = new Scanner (System.in);
+
+    // declare a variable to store the first integer
+    System.out.println("Please enter an integer.");
+    //prompt the user for the first integer 
+    int first_integer = input.nextInt();
+    System.out.println("Please enter another integer.");
+    //declare a variable to store second_integer
+    //prompt the user for the second integer
+    int second_integer = input.nextInt();
```

Comment:
Remember, variables should always be named using camelCase and describe what they are storing or being used for. You are using a different format that would typically be used in other languages (Python for instance). For Java, we tend to use camelCase instead of snake_case.